test/features/steps/crud_database.py

```diff
@@ -85,6 +93,16 @@ def step_see_db_dropped(context):
     wrappers.expect_exact(context, 'Query OK, 0 rows affected', timeout=2)
 
 
+@then('we see database dropped and no default database')
+def step_see_db_dropped_no_default(context):
+    """Wait to see drop database output."""
+    user = context.conf['user']
+    host = context.conf['host']
+    context.currentdb = '(none)'
```

Comment:
Maybe `context.currentdb = None` might be better (I'm not sure). dbname=None seem to be used in other places in the behave tests.